server/portal/creation.py

```diff
@@ -176,6 +208,16 @@ def save_image():
     return jsonify({'image_url': aws_url})
 
 
+"""
+Endpoint: /portal/post/approve
+HTTP Methods: POST
+Response Formats: JSON
+Content-Type: application/x-www-form-urlencoded 
```

Comment:
Remove trailing whitespace and fix all other Flake8 errors. Can be found here: https://circleci.com/gh/pennlabs/labs-api-server/37?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link